kubernetes/resource_kubernetes_horizontal_pod_autoscaler_v2beta2.go

```diff
@@ -53,6 +53,7 @@ func resourceKubernetesHorizontalPodAutoscalerV2Beta2() *schema.Resource {
 						"behavior": {
 							Type:        schema.TypeList,
 							Description: "Behavior configures the scaling behavior of the target in both Up and Down directions (scale_up and scale_down fields respectively).",
+							Computed:    true,
```

Comment:
How come this needs to be computed?

Reply:
While I was working on this PR I noticed that terraform apply constantly produces a plan with changes related to the `behavior` attribute. Kubernetes uses default HPA Scaling rules if they are not set and the default values of these rules are part of the API response. That updates the state file with the `behavior` attribute after applying and causes the attribute deletion during the next plan.

Kubernetes documentation [link](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#horizontalpodautoscaler-v2beta2-autoscaling): _`behavior` ... If not set, the default HPAScalingRules for scale up and scale down are used._

You are right, it doesn't relate to the aim of this PR and shouldn't be here. Removed, thank you!